pkg/permissions/permissions.go

```diff
@@ -348,6 +348,30 @@ func createAppSet(db prefixer.Prefixer, typ, docType, slug string, set Set) (*Pe
 	return doc, nil
 }
 
+// MergeExtraPermissions merges rules from "extraPermissions" set by adding them
+// in the "perms" one
+func MergeExtraPermissions(perms, extraPermissions Set) (Set, error) {
+	var err error
+	var mergedRule *Rule
+	var permissions Set
+
+	for _, rule := range perms {
+		for _, newRule := range extraPermissions {
+			if rule.Title == newRule.Title {
+				mergedRule, err = rule.Merge(newRule)
```

Comment:
[nit] I think it would be better to declare the two variables here (`mergedRule, err := rule.Merge(newRule)`)